sqlmesh/utils/jinja.py

```diff
@@ -358,6 +359,26 @@ def to_expressions(self) -> t.List[Expression]:
 
         return output
 
+    @property
+    def data_hash_values(self) -> t.List[str]:
+        data = []
+
+        for macro_name, macro in sorted(self.root_macros.items()):
+            data.append(macro_name)
+            data.append(macro.definition)
+
+        for _, package in sorted(self.packages.items(), key=lambda x: x[0]):
```

Comment:
you don't need to pass in key right, can't you just do 

sorted({"x": 1}.items())

lambda x: x[0] is effectively doing the same thing?

Reply:
I think you're right